samples/tiffaddpath-test.cpp

```diff
@@ -32,7 +32,7 @@ struct TiffTagInfo {
     const char* name_;
 };
 
-extern const TiffTagInfo tiffTagInfo[] = {
+constexpr TiffTagInfo tiffTagInfo[] = {
```

Comment:
Honestly, I do not understand the benefits of these changes. Could you please give some details about it? 😉